src/test/Makefile

```
@@ -16,8 +16,9 @@ TEST_DIR = unit
 ROOT = ../..
 OBJECT_DIR = ../../obj/test
```

Ah your push got away with 
```suggestion
OBJECT_DIR = $(ROOT)/obj/test
```

---

Oops, will update my commit